tectonic/dpx-dvipdfmx.c

```diff
@@ -757,10 +713,6 @@ dvipdfmx_main (int argc, char *argv[])
   pdf_load_fontmap_file("kanjix.map", FONTMAP_RMODE_APPEND);
   pdf_load_fontmap_file("ckx.map", FONTMAP_RMODE_APPEND);
 
-  /* End config file fakery. */
-
-  do_args (argc, argv, NULL, 0);
```

Comment:
Can the `do_args` function now be removed? I'm not seeing it show up in the diff.